### PR TITLE
Less computation on notebooks

### PR DESCRIPTION
**Pull request by RondeauG**
### Pull Request Checklist:
- [x] This PR addresses an already opened issue (for bug fixes / features)
  - This PR fixes #xyz
- [x] (If applicable) Documentation has been added / updated (for bug fixes / features).
- [x] (If applicable) Tests have been added.
- [x] CHANGELOG.rst has been updated (with summary of main changes).
  - [x] Link to issue (:issue:`number`) and pull request (:pull:`number`) has been added.

### What kind of change does this PR introduce?

* Reduces the computation needed by the notebooks. Notably:
  - The modelling notebook now gets its data from `xhydro-testdata` instead of using `xdatasets/planetary-computer`.
  - The local frequency analysis notebook is done on 2 stations instead of 5.
  - The regional analysis notebook is done on 15 stations instead of 25. Only annual maximums are used.
  - The uncertainty analysis is done using 35 bootstraps instead of 200.

### CODE REVIEW

**Comment by RondeauG** [comment]
@sebastienlanglois Aurais-tu une idée de ce qui plante actuellement dans le module `gis`? Tout passait correctement il y a trois jours (e.g. https://github.com/hydrologie/xhydro/pull/209) et je vois très peu de changements dans les environnements. J'ai fait quelques tests localement, mais je ne trouve pas...

**Comment by sebastienlanglois** [comment]
Oui, il semble que le Planetary Computer ont bloqué l'accès à leur stockage via http depuis hier. Il existe un fix où il faut demander un SAS token afin que l'url soit sécurisé, mais a priori avec cette approche, ça devrait réparer le module. Je vais implémenter un fix d'ici demain et pousser une PR.

**Comment by Zeitsperre** [comment]
@sebastienlanglois If we want to add a token to the repository secrets as an environment variable, this can be easily done for both GitHub and ReadTheDocs. Contact me if you're interested in that approach.

**Comment by sebastienlanglois** [comment]
@Zeitsperre @RondeauG I have implemented the fix for the GIS module here : https://github.com/hydrologie/xhydro/pull/213

File: docs/notebooks/hydrological_modelling.ipynb
Comment (Zeitsperre) [inline docs/notebooks/hydrological_modelling.ipynb:98]: <p>Once #212 is merged, we can replace this code with:</p>

<pre class="ql-syntax" spellcheck="false">from xhydro.testing.helpers import deveraux
import xarray as xr

meteo_file = "hydro_modelling/ERA5_testdata.nc"
ds = xr.open_dataset(deveraux.fetch(meteo_file))
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/hydrologie/xhydro/pull/211/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

**Review by Zeitsperre — APPROVED** [review]
LGTM, but wait for @sebastienlanglois's approval

File: docs/notebooks/hydrological_modelling.ipynb
Comment (sebastienlanglois) [inline docs/notebooks/hydrological_modelling.ipynb:1]: Is the `import planetary_computer` still required ?

**Review by sebastienlanglois — COMMENTED** [review]
LGTM

**Review by sebastienlanglois — APPROVED** [review]

File: docs/notebooks/hydrological_modelling.ipynb
Comment (Zeitsperre) [reply docs/notebooks/hydrological_modelling.ipynb:96]: Leave this for now. The other branch will add this change in!

**Pull request merged**
